docs/mdbook.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e; set -u
+
+# Setup the the mdbook binaries if they havent already been setup
```

Comment:
Can we install with cargo?

Reply:
It needs to run on netlify so unfortunately not.

Reply:
Can we do the checks on our CI instead?

Reply:
Doing that would mean we have two locations that we need to manage running mdbook.
Surely doing it all in one place would be simpler?

If we build both mdbook and mdbook-linkcheck on CI that will make CI run longer and use more of our cache.

Reply:
There are some basic syntax checks that netlify is already handling for us.